src/components/graph/graph.ts

```diff
@@ -444,13 +439,6 @@ export default class GraphComponent {
                 .attr('y', d.y + 15)
           })
 
-        this._nodes.each((n: IGraphNodeData, i_: number, refs_: any[]) => {
```

Comment:
i removed it because we already set these attributes above this so why are we setting them again?

Reply:
This breaks dragging when you drag on the text to drag a node. So what's happening is up at the start of this 'drag' handler, we're not checking if the dragged element was a text el or a circle el. If we can somehow check which one it is, I don't know how, we can then maybe simplify this further and reduce doing duplicate work. Right now it assumes it's a circle, then it checks if the nodeTextLabel's is not null, then it'll update the text positions, then update the circle positions again sometimes for no reason since it did update it already. In other cases, it will update the text positions twice.

Perhaps, maybe in a different PR once this is merged, we can apply a transform to the whole node group instead of applying a translation to the node and text label separately. It would most definitely be faster than doing it for each individually.